setup.py

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
-from setuptools import setup
+from setuptools import find_packages, setup
 
 from djangocms_googlemap import __version__
 
+
+REQUIREMENTS = [
+    'django-cms>=3.3.1',
```

Comment:
don't require 3.3.1 since you already included the related_name fix.
You can support >= 3.3.0